contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -889,3 +890,19 @@ get_pltsql_function_signature(PG_FUNCTION_ARGS)
 	ReleaseSysCache(proctup);
 	PG_RETURN_TEXT_P(cstring_to_text(func_signature));
 }
+
+void init_and_check_common_utility(void)
+{
+	if (!common_utility_plugin_ptr)
+	{
+		common_utility_plugin **utility_plugin;
+		utility_plugin = (common_utility_plugin **) find_rendezvous_variable("common_utility_plugin"); 
+		common_utility_plugin_ptr = *utility_plugin;
+
+		/* common_utility_plugin_ptr is still not initialised */
+		if (!common_utility_plugin_ptr)
+			ereport(ERROR,
+				(errcode(ERRCODE_INTERNAL_ERROR),
+				 errmsg("common utility plugin pointer is not initialised properly.")));
```

Comment:
Give a better error like "Failed to find common utility plugin"

Reply:
Updated!